pkgs/applications/audio/callaudiod/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkg-config
+, glib
+, alsaLib
+, libpulseaudio
+}:
+
+stdenv.mkDerivation rec {
+  pname = "callaudiod";
+  version = "0.0.4";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.com";
+    owner = "mobian1";
+    repo = pname;
+    rev = version;
+    sha256 = "07k7xp5a9c4d4lq7amaj6cg6b3gsd77x9wvf7nzcf4vpaph4yiyj";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    alsaLib
+    libpulseaudio
+    glib
+  ];
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```
This got recently deprecated and shouldn't be used for new packages.